doc/release/1.17.0-notes.rst

```diff
@@ -42,6 +42,18 @@ In the future it will not be possible to switch the writeable flag to ``True``
 from python.
 This deprecation should not affect many users since arrays created in such
 a manner are very rare in practice and only available through the NumPy C-API.
+Writing to the result of `numpy.broadcast_array` will warn
+----------------------------------------------------------
+
+Commonly `numpy.broadcast_array` returns a writeable array with internal
+overlap, making it unsafe to write to. A future version will set the
+``writeable`` flag ``False``, and require users to manually set it ``True`` if
```

Comment:
set *to* `False` and set it *to* `True`.

Reply:
adopted and re-flowed the paragraph